README.md

```diff
@@ -418,6 +418,7 @@ AS
   SELECT * FROM get_pg_stat_statements();
 
 GRANT SELECT ON postgres_exporter.pg_stat_statements TO postgres_exporter;
+GRANT ALL ON FUNCTION pg_catalog.pg_ls_waldir() TO postgres_exporter;
```

Comment:
I always hesitate to recommend someone give ALL permissions. I think this only needs `EXECUTE`. Can you confirm and update?

Reply:
I ran on this comment by chance and did it on my side as I had the error, and it did clear the error from the console when granting only EXECUTE. So I guess it does work :+1: